package.json

```
@@ -29,8 +29,8 @@
   },
   "dependencies": {
     "@bdelab/roam-apps": "1.2.8",
-    "@bdelab/roar-firekit": "^9.7.1",
-    "@bdelab/roar-letter": "2.1.0",
+    "@bdelab/roar-firekit": "^9.8.0",
+    "@bdelab/roar-letter": "3.0.1",
```

Roar letter should not be updated to 3.0.1 yet
```suggestion
    "@bdelab/roar-letter": "2.1.0",
```